gemfiles/rails61.gemfile

```diff
@@ -9,7 +9,7 @@ gem "rails", "~> 6.1.0"
 gem "carrierwave", "~> 2"
 gem "combustion", ">= 1.3"
 gem "rbnacl", ">= 6"
-gem "sqlite3", "~> 1.4"
+gem "sqlite3", "< 2"
```

Comment:
The upstream introduced this constraint. It's essentially the same as `~> 1.4` for our intents and purposes, so decided to keep what the upstream had to minimize the diff.